src/net_processing.cpp

```diff
@@ -519,6 +519,7 @@ class PeerManagerImpl final : public PeerManager
     std::optional<std::string> FetchBlock(NodeId peer_id, const CBlockIndex& block_index) override
         EXCLUSIVE_LOCKS_REQUIRED(!m_peer_mutex);
     bool GetNodeStateStats(NodeId nodeid, CNodeStateStats& stats) const override EXCLUSIVE_LOCKS_REQUIRED(!m_peer_mutex);
+    std::vector<TxOrphanage::OrphanTxBase> GetOrphans() override EXCLUSIVE_LOCKS_REQUIRED(!m_tx_download_mutex);
```

Comment:
Perhaps `GetOrphanTransactions`, since this is on the peerman and orphan blocks can exist too

Reply:
Agreed. Updated.